.gitignore

```diff
@@ -1,27 +1,43 @@
+# Please keep workflow- and tool-specific patters out of this
+# file. Just put patterns here that are very generic and we want to
+# avoid adding by mistake and files that are generated by the build
+# system.
+# 
+# Comments from gitignore(1) mention that which file to place a
+# pattern in depends on how the pattern is meant to be used:
+#
+# * Patterns which should be version-controlled and distributed to
```

Comment:
I don't think it is helpful to see this information as dogma. Besides, I think it is up to us to define which patterns should be "distributed to other repositories via clone" and which ones "do not need to be shared with other repositories."